external-crates/move/crates/move-compiler/tests/move_check/parser/unmatched_restricted_identifier.move

```diff
@@ -1,5 +1,5 @@
 // unmatched `
 
 module a::m {
-    fun foo(`x: u64): u64 { `x }
+    fun foo(`x: u64) { `x }
```

Comment:
```suggestion
    fun foo(`x: u64): u64 { `x }
```

Error casing should help with this.

Reply:
I'm confused: will this place `UnexpectedError` in the body? If os, the type annotation should be just fine.

Reply:
+1 here, I would expect an error in the body

Reply:
> +1 here, I would expect an error in the body

Which we now have so I resurrected the returned type